react/package.json

```diff
@@ -1,8 +1,35 @@
 {
   "name": "vtex.render-getting-started",
   "version": "0.0.0",
+  "scripts": {
+    "test": "jest",
+    "update-snapshots": "jest --updateSnapshot"
```

Comment:
I suppose you could remove this script, the user can update the snapshots using `npm test -- -u`